enterprise_access/apps/api/v1/urls.py

```diff
@@ -39,6 +39,7 @@
 # BFFs
 router.register('bffs/learner', views.LearnerPortalBFFViewSet, 'learner-portal-bff')
 router.register('bffs/health', views.PingViewSet, 'bff-health')
+router.register('bffs/checkout', views.CheckoutBFFViewSet, 'checkout-bff')
```

Comment:
Sorry one last request: Can you protect this behind `settings.ENABLE_CUSTOMER_BILLING_API`? Or pick a new name for the endpoints under development for SSP.